test/blackhole-server.c

```diff
@@ -43,20 +43,20 @@ static void connection_cb(uv_stream_t* stream, int status) {
   conn_rec* conn;
   int r;
 
-  ASSERT(status == 0);
-  ASSERT(stream == (uv_stream_t*)&tcp_server);
+  ASSERT_EQ(status, 0);
+  ASSERT_EQ(stream, (uv_stream_t*)&tcp_server);
```

Comment:
I think these need to use `ASSERT_PTR_EQ`, otherwise I expect we'll get a lot of warnings on 32-bit